### ISSUE TITLE
[Bug]: Incorrect aria-label implementation on igc-slider component

### ISSUE THREAD

**Issue by georgianastasov** [IgniteUI/igniteui-webcomponents#1410]
### Which component(s) are affected?

Slider (single/range)

### Description

There is an issue (enhancement) with the igc-slider component where the aria-label attribute is applied to both the igc-slider and its inner thumb element. This is not correct because the aria-label should be applied only to the thumb, which is the focusable part of the slider. When navigating via keyboard (tabbing), users focus directly on the thumb and not the slider container itself.

### Reproduction

To observe the issue:

    1. Include an igc-slider component in your project.
    2. Inspect the rendered HTML in the browser.
    3. Notice that both the igc-slider and its inner thumb element have the aria-label attribute applied.

### Workaround

I have not found a workaround.

### Is this a regression?

No or unsure. This never worker, or I haven't tried before.

### Affected versions

5.0.2

### Browser/OS/Node environment

Browser: All modern browsers (e.g., Chrome, Firefox, Edge)
Operating System: All platforms (e.g., Windows, macOS, Linux)

**Comment by simeonoff** [comment]
@SisIvanova we will have to defer this fix as after some discussion with @rkaraivanov, we would like to implement a different solution that will solve this and other issues in the component.

**Comment by simeonoff** [comment]
@rkaraivanov was [referenceTarget](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/reference-target-explainer.md) what we were waiting for to be able to properly close this issue?
